src/lidar/read_hokuyo_lidar.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "../simulator/utils.h"
+
+namespace lidar
+{
+	bool initializeLidar();
+	bool isLidarInitialized();
```

Comment:
I don't see an implementation for this method anywhere. We can probably just delete this declaration